doc/user/quick-sql-grammar.md

```diff
@@ -79,6 +81,22 @@
 | /unique                                 | Generate table level unique constraint |
 <!-- markdownlint-enable MD013 -->
 
+### Star/Snowflake schema relationship direction indicators
+
+In star (and snowflake) database model tables are organized into a hierarchy where the table at the top (fact) is in many-to-one relationship to children (dimension tables). Consequently, each dimension table is equipped with primary key, which is referenced via foreign key constraint in the fact table. For example, given the Sales fact table together with Products and Customers dimension tables, the star database model is expressed in QSQL as:
```

Comment:
- [ ] Use an 80 characters rule, instead of a very long line
- [ ] Change QSQL for Quick SQL (Two words)